src/core/wall.ts

```diff
@@ -43,4 +56,30 @@ export const MapCatalog: StoredWallState[][] = [[
   {pos: {x: 205, y: 470}, width: 70, height: 140, sprite: WallSprite.BUNKER_1x2_1},
   {pos: {x: 650, y: 505}, width: 140, height: 70, sprite: WallSprite.BUNKER_2x1_2},
   {pos: {x: 685, y: 110}, width: 70, height: 140, sprite: WallSprite.BUNKER_1x2_1},
+  {pos: {x: 0, y: 650}, width: 140, height: 70, sprite: WallSprite.BUNKER_2x1_2},
```

Comment:
This'd be better as 4 walls automatically created for any map that stretch the entire height/width and exist at the boundaries of the world (so negative/overflowing positions so they wouldn't be visible, but would still be checked by the collision logic)